kani-driver/src/args.rs

```diff
@@ -83,6 +83,11 @@ pub struct KaniArgs {
     #[structopt(flatten)]
     pub checks: CheckArgs,
 
+    /// Zero initilize variables.
+    /// This is an unstable feature.
```

Comment:
Add a comment mentioning the performance vs soundness tradeoff?

Reply:
Done